docs/superpowers/specs/2026-04-10-tutorial-guide-design.md

```
@@ -0,0 +1,141 @@
+| 순서 | 대상 위젯 | 설명 텍스트 |
+|-----|----------|-----------|
+| 1 | 지도 | "게임 맵이에요. 내 위치와 구역이 표시돼요" |
+| 2 | 타이머 | "남은 게임 시간이에요" |
+| 3 | 참가자 목록 | "현재 참가자 상태를 확인할 수 있어요" |
+| 4 | 채팅 버튼 | "팀원과 채팅할 수 있어요" |
+| 5 | 체포/탈옥 버튼 | "경찰은 체포, 도둑은 탈옥할 때 사용해요" |
+| 6 | QR 스캔 | "상대방의 QR을 스캔하여 체포/탈옥해요" |
+
+## 파일 구조
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)